src/software/thunderscope/gl/layers/BUILD

```diff
@@ -157,3 +157,13 @@ py_library(
         requirement("pyqtgraph"),
     ],
 )
+
+py_library(
+    name = "gl_max_dribble_layer",
+    srcs = ["gl_max_dribble_layer.py"],
+    deps = [
+        ":gl_layer",
+        "//software/thunderscope/gl/graphics:gl_polygon",
```

Comment:
I believe you need to add `software/thunderscope:util` in this `BUILD` file. That seems to be why the simulated tests fail in CI